docs/functions/sys.html

```diff
@@ -22,13 +22,6 @@ <h5>sys.is64</h5>
 	<code>sys.is64</code>
 </section>
 
-<section id="sys.dark" class="my-5">
-	<h5>sys.dark</h5>
-	<p>Check if dark mode is enabled.</p>
-	<p>Syntax</p>
-	<code>sys.dark</code>
-</section>
-

```

Comment:
Can be retained. See https://github.com/moudey/Shell/pull/481#issuecomment-2220988856